test/integration_test_helper.rb

```diff
@@ -36,6 +36,15 @@ def kube_server_version
       _kubectl.server_version
     end
 
+    def server_dry_run_available?
```

Comment:
Can we just assume its available if we're on version 1.13 or greater?

Reply:
Yes, after 1.13, this feature is enable by default.

Reply:
I mean can this code just check that the server is on version >= 1.13 ?

Reply:
We could do that. Since the possibility of people disable server-dry-run feature when >=1.13 is pretty low. 
Sure, let me change to the version check.